### DISCUSSION TITLE
Terminal

### DISCUSSION THREAD

**Discussion by Amarachi199** [doocs/advanced-java#233] [Q&A]
Greetings. 
Please I am new to java and I am stuck at terminal stage! I have been stuck there for three days. I use Windows 7. ![image](https://user-images.githubusercontent.com/88140837/127477781-95c4a23c-67e8-401d-b10a-e3d3b5401c2c.jpeg)[Uploading 127477581...]

**Comment by Jlc420z** [comment]
[google](url)
